app/lib/send_automated_message.rb

```diff
@@ -0,0 +1,66 @@
+class SendAutomatedMessage
+  attr_accessor :locale, :message, :message_name, :tax_return, :locale, :message_tracker, :sent_messages, :client, :user
+
+  def initialize(client:, message:, tax_return: nil, locale: nil, message_name: nil)
+    @client = client
+    @locale = locale || client.intake.locale || "en"
```

Comment:
This allows an explicitly passed locale, but then falls back to the intake locale and then english if its not passed.

I feel like it should be sufficient to use the intake locale or english, but since the original ClientMessageService this is extracted from takes an explicit locale, I wasn't sure if I could safely remove it at this point.